Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -76,6 +79,28 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
                 return "Simulated a tunnel error as requested"
             }
         }
+
+        var errorCode: Int {
+            switch self {
+                // Tunnel Setup Errors - 0+
+            case .startingTunnelWithoutAuthToken: return 0
```

Comment:
Do we need a separate error for both waitlist and sub users, since iOS and Mac App Store try to read from the subscription token store directly? It would be useful to know for example which token the extension tried and failed to fetch.

Reply:
As discussed privately, this may be non-trivial right now so I'm proceeding without addressing this.

Also the VPN menu app should no longer try to start the tunnel without an auth token.